.github/workflows/charm-release.yaml

```diff
@@ -57,6 +57,17 @@ jobs:
       - name: Unpack Artifact
         if: ${{ inputs.artifact != '' }}
         run: sudo apt-get update && sudo apt-get install tar && tar xf artifact.tar.gz
+      - name: Initialize lxd  # This should dropped once it's implemented on charming-actions itself.
+        if: ${{ inputs.destructive-mode == 'false' }}
+        run: |
+          sudo lxd init --auto
+          sudo chmod a+wr /var/snap/lxd/common/lxd/unix.socket
+          lxc network set lxdbr0 ipv6.address none
+          sudo usermod -a -G lxd $USER
+          if [[ $(cat /etc/os-release | grep VERSION_CODENAME) == *"jammy"* ]]; then
+            sudo iptables -F FORWARD
+            sudo iptables -P FORWARD ACCEPT
+          fi
```

Comment:
Could we also leave some comments or links to some documentation on why we have the additional steps after `lxd initi --auto`? It'll be nice to know the reason behind the changes.

Reply:
I added a small comment. 
Reference was taken from [here](https://github.com/charmed-kubernetes/actions-operator/blob/df660b2ba29176d8b638e51c40240b4bc9bd58c4/src/bootstrap/index.ts#L222).
More information is available [here](https://documentation.ubuntu.com/lxd/en/latest/explanation/networks/). (Though I am also yet to read all of it.)